Apps/Patient/src/Models/DiagnosticImagingStatus.cs

```diff
@@ -50,5 +50,10 @@ public enum DiagnosticImagingStatus
         /// Exam is amended.
         /// </summary>
         Amended,
+
+        /// <summary>
+        /// Exam is final.
+        /// </summary>
+        Final,
```

Comment:
We can remove the unused enum members: InProgress and Completed.

Reply:
Removed @MikeLyttle